### DISCUSSION TITLE
module 'duckdb.experimental.spark.sql.functions' has no attribute 'concat_ws'Feature Request: ...

### DISCUSSION THREAD

**Discussion by nicornk** [duckdb/duckdb#11044] [Ideas]
Why do you want this feature?

Spark compatibility

**Comment by Tishj** [comment]
We do have `concat_ws` in DuckDB, you're welcome to submit a PR to make it available through the spark API

<https://duckdb.org/docs/sql/functions/char>

<https://github.com/duckdb/duckdb/blob/main/tools/pythonpkg/duckdb/experimental/spark/sql/functions.py>

**Reply by Tishj** [reply]
I think it's as easy as replicating the behaviour that other functions like https://github.com/duckdb/duckdb/blob/953959558c125ebd1c0d91e02f29155f772e61a5/tools/pythonpkg/duckdb/experimental/spark/sql/functions.py#L84 use, and adding a test for it

**Reply by nicornk** [reply]
Thanks for the pointers, I'll see what I can do!

**Comment by nicornk** [comment]
PR: https://github.com/duckdb/duckdb/pull/11051
